docs/articles/configs/toolchains.md

```diff
@@ -5,65 +5,103 @@ name: Toolchains
 
-When you run your benchmarks without specifying the toolchain in an explicit way, we use the default one. It works OOTB, you don't need to worry about anything.
+When you run your benchmarks without specifying the toolchain in an explicit way, the default one is used:
 
-We use Roslyn for classic .NET and Mono, and `dotnet cli` for .NET Core and CoreRT.
+* Roslyn for Full .NET Framework and Mono
+* dotnet cli for .NET Core and CoreRT
 
 ## Multiple frameworks support
 
-You can target multiple frameworks with single, modern csproj file:
+
+If you want to test multiple frameworks, your project file **MUST target all of them** and you **MUST install the corresponding SDKs**:
 
 ```xml
-<TargetFrameworks>netcoreapp2.0;net462</TargetFrameworks>
+<TargetFrameworks>netcoreapp3.0;netcoreapp2.1;net48</TargetFrameworks>
+```
+
+If you run your benchmarks without specifying any custom settings, BenchmarkDotNet is going to run the benchmarks **using the same framework as the host process**:
+
+```cmd
+dotnet run -c Release -f netcoreapp2.1 # is going to run the benchmarks using .NET Core 2.1
+dotnet run -c Release -f netcoreapp3.0 # is going to run the benchmarks using .NET Core 3.0
+dotnet run -c Release -f net48         # is going to run the benchmarks using .NET 4.8
+mono $pathToExe                        # is going to run the benchmarks using Mono from your PATH
+```
+
+To run the benchmarks for multiple runtimes with a single command, you need to specify the target framework moniker names via `--runtimes|-r` console argument:
+
+```cmd
+dotnet run -c Release -f netcoreapp2.1 --runtimes netcoreapp2.1 netcoreapp3.0 # is going to run the benchmarks using .NET Core 2.1 and .NET Core 3.0
+dotnet run -c Release -f netcoreapp2.1 --runtimes netcoreapp2.1 net48         # is going to run the benchmarks using .NET Core 2.1 and .NET 4.8
+```
+
+What is going to happen if you provide multiple Full .NET Framework monikers? Let's say:
+
+```cmd
+dotnet run -c Release -f net461 net472 net48
 ```
 
-BenchmarkDotNet allows you to take full advantage of that. With single config, we can execute the benchmarks for all the frameworks that you have listed in your csproj file.
+Full .NET Framework always runs every .NET executable using the latest .NET Framework available on a given machine. If you try to run the benchmarks for a few .NET TFMs, they are all going to be executed using the latest .NET Framework from your machine. The only difference is that they are all going to have different features enabled depending on target version they were compiled for. You can read more about this [here](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/version-compatibility) and [here](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/application-compatibility). This is **Windows Operating System behavior which can not be controlled by BenchmarkDotNet or any other tool**.
```

Comment:
This is not technically anything to do with Windows - it has to do with how we choose to ship and package .NET Framework. (Indeed, we had a TFM for .NET Framework 2.0 or 1.1 it would load that runtime if it were installed.)